content/community/governance/gcb/index.md

```diff
@@ -0,0 +1,102 @@
+# Galaxy Community Board Proposal
+
+**Keywords:** Special Interest Groups, Community of Practices
+
+## Origins
+
+The Galaxy Community Board (GCB) is proposed to take over the functions of the Global Galaxy Steering Committee.
+During GCC 2023 we discussed how to empower the Galaxy community, in terms of Project planning and effective execution, decision making and governance. 
+
+## Purposes
+
+Representation for community aspects of the ecosystem is central to long term Project sustainability. The goal is to empower participants, giving a genuine voice, in planning and governance, through efficient and transparent structures and processes, that can scale effectively as the Galaxy project continues to expand. 
+
+The GCB will provide a supportive virtual forum for the exchange of ideas, and a governance body representing the interests of participating communities. It will organise and manage its own agenda and work, including arranging meetings and asynchronous discussions. The agenda might include planning new initiatives; supporting ongoing development; identifying and advocating for user needs; managing coherent, community owned proposals and responses in the roadmap processes and other important Project decision making.
+
+Efficient communication between the GCB (Community Board) and both the GEB (Executive Board) and the GTB (Technical Board) requires at least one representative from GEB and from GTB to be also members of the GCB. It is recommended that at least one member of each of the other Boards are also members of the GCB, at all times.
```

Comment:
The second sentence reads like a replica of the first sentence's content (except the trailing 'at all times'). Or are there more boards than GEB and GTB intended to be represented in the GCB?

Reply:
Ok - the recommendation just follows the requirement quite closely/obviously. Adding "Therefore, it is" will sufficiently resolve this. Maybe "of each of the other *two* Boards is another clarification.